debug_toolbar/static/debug_toolbar/js/history.js

```diff
@@ -104,3 +104,4 @@ $$.on(djDebug, "click", ".refreshHistory", function (event) {
     event.preventDefault();
     refreshHistory();
 });
+$$.onPanelRender(djDebug, "HistoryPanel", refreshHistory);
```

Comment:
Can you add a comment for this change? I'm not sure why we need this.

Reply:
Done. We need this because the en ajax request occurs we don't necessary refresh the toolbar. In the first load on the history panel we haven't problem, but after the list don't auto udpate. We can click on refresh button but I think it's better to update the list each render of history.